runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -50,17 +52,18 @@ public class Restore {
     private String restoreTempDirPath;
 
     // The filename of each table's backup file, format: uuid.namespace$tableName.
-    private final List<String> tableBackups;
+    private List<String> tableBackups;
 
     private CorfuRuntime rt;
 
     // The Corfu Store associated with the runtime
     private CorfuStore corfuStore;
 
-    // If FULL, will clean up all tables before restore;
-    // If PARTIAL, will only clean up tables that are to be restored
+    // Restore mode. Refer to class definition for details
     private RestoreMode restoreMode;
 
+    private static final String CORFU_SYSTEM_NAMESPACE = "CorfuSystem";
```

Comment:
nit: can we use the definition from TableRegistry instead of redefining the string here?

Reply:
Done.